### PR TITLE
Fixed large count of notices

### PR DESCRIPTION
**Pull request by dionisvl**

### CODE REVIEW

**Review by williamdes — APPROVED** [review]
This looks good, thank you
I will have a look before merging.

File: tcpdf.php [hunk 1]
```diff
@@ -2735,20 +2739,42 @@ protected function adjustCellPadding($brd = 0) {
 				}
 			}
 			// correct internal cell padding if required to avoid overlap between text and lines
-			if ((strpos($border, 'T') !== false) and ($this->cell_padding['T'] < $adj)) {
+			if (
+				is_numeric($this->cell_padding['T'])
```
Comment (mvorisek) [inline tcpdf.php:2743]: when not numeric, it is still a valid input?

Comment (williamdes) [reply tcpdf.php:2743]: Do you want to be a project PR reviewer ?

Comment (mvorisek) [reply tcpdf.php:2743]: Maintainer permission will be good, I already maintain `atk4/*` repos, but I can not make any promises due to my resources.

Comment (dionisvl) [reply tcpdf.php:2743]: > when not numeric, it is still a valid input?

I don't think so. You can see string #225 about it.

Comment (mvorisek) [reply tcpdf.php:2743]: Then we should throw instead or keep php emitting notice warning (ie. your usage of this lib should be fixed) :)

Comment (mvorisek) [reply tcpdf.php:2743]: > 
> 
> Do you want to be a project PR reviewer @mvorisek ?

please add also Maintainer perm to be able to merge PR and edit the PR/issues content/discussions

Comment (williamdes) [reply tcpdf.php:2743]: > please add also Maintainer perm to be able to merge PR and edit the PR/issues content/discussions

Sorry but I need to earn trust to allow people to merge work without me approving.
You have the grants for the issues and PR edits, open and closing :)

Comment (mvorisek) [reply tcpdf.php:2743]: Hmm, not sure what is the advantage then... :) I belive any maintainer should have at least the ability to participate on non protected branched and full discussion edit (I use it a lot to correct main issue/PR desc/title).

Merge access is not needed for me and can be configured by requiring more than 2 maintainer approvals to allow merge into protected branches.

Comment (williamdes) [reply tcpdf.php:2743]: > Hmm, not sure what is the advantage then... :) I belive any maintainer should have at least the ability to participate on non protected branched and full discussion edit (I use it a lot to correct main issue/PR desc/title).

I understand, for now I can not tweak the access level further. Have to search on the GUIs to find out.

One advantage is using codeowners file to trigger reviews on PRs.

You should at least be able to edit PR milestones

Hi @Guileas 
Do you have the same issue ?

image

Comment (mvorisek) [reply tcpdf.php:2743]: > One advantage is using codeowners file to trigger reviews on PRs.

first thing I disabled :)

> You should at least be able to edit PR milestones

I can edit them, but still no discussion nor title/desc

Comment (Guileas) [reply tcpdf.php:2743]: @williamdes yeah same issue

Comment (williamdes) [reply tcpdf.php:2743]: Should be better now, I moved to write and protected all the possible branches. Please keep using forks ;)

Comment (mvorisek) [reply tcpdf.php:2743]: Now good :D, feel free to delete the unrelated discussion here ;-)

**Comment by mvorisek** [comment]
fixes in tcpdf upstream

**Comment by williamdes** [comment]
And latest upstream is merged here

**Pull request closed without merging**